draft-beky-httpbis-metadata.md

```diff
@@ -110,12 +113,21 @@ that receives METADATA for a stream in the “idle” state _MAY_ choose to reta
 the payload for a period of time, under the assumption that the stream will soon
 transition to the “open” state.
 
+A metadata block is the concatenation of the payloads of a sequence of one or
+more METADATA frames, among which only the last one carries the END_METADATA
+flag.   If the transfer of the last metadata block cannot be completed due to
+the stream or connection being closed before a METADATA frame with the
+END_METADATA flag, then the metadata block fragment carried by the last run of
+METADATA frames without the END_METADATA flag _SHOULD_ be discarded.  This
```

Comment:
optional:

"..., then the metadata block fragment carried by the last run of METADATA frames without the END_METADATA flag"

-->

"..., then that incomplete metadata block"

(I think this shortening is okay because the first sentence defines a metadata block very clearly. Nice!)

Reply:
Done, expect "the" instead of "that" seems to flow a little better to me, without introducing ambiguity.